### ISSUE TITLE
What is the graphic memory size during training？ and how long did the training process take?

### ISSUE THREAD

**Issue by Bian-666** [yihongXU/TransCenter#16]
I'm interested in your work. but i only have a single rtx3090 machine.

**Comment by yihongXU** [comment]
With the efficient version, you can train the model with rtx3090 with a batch size of 2, we were using RTX Titan with the same memory size, it did work (**torch.backends.cudnn.benchmark=True** can further reduce the memeory footprint for some gpus). If you want to have a bigger batch size, maybe try with [gradient accumulation](https://kozodoi.me/python/deep%20learning/pytorch/tutorial/2021/02/19/gradient-accumulation.html#:~:text=Simply%20speaking%2C%20gradient%20accumulation%20means,might%20find%20this%20tutorial%20useful.) . 

Another solution would be use another PVT_v2 backbone. We used B2 for TransCenter(-Dual) and B0 for TransCenter-Lite. You have other options from [B0 to B5](https://github.com/whai362/PVT).

Have fun!

**Comment by Bian-666** [comment]
Thank you for your prompt reply，thanks for your great work!
